.github/workflows/lint.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   luacheck:
     name: Luacheck
     # The type of runner that the job will run on
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Lint step failed due to deprecation of ubuntu-20.04
![image](https://github.com/user-attachments/assets/be935e13-ef9f-47a8-9b78-25bd79bbb0d6)
https://github.com/Almo7aya/openingh.nvim/actions/runs/14692342181/job/41229557982
https://github.com/actions/runner-images/issues/11101